spec/models/organization_sync_spec.rb

```diff
@@ -10,6 +10,28 @@
       it '#execute changes state from created to executing' do
         expect { subject.execute }.to change(subject, :state).from('created').to('executing')
       end
+
+      context '#execute completed' do
+        before do
+          subject.execute
+          subject.complete
+        end
+
+        it '#execute changes state from completed to executing' do
+          expect { subject.execute }.to change(subject, :state).from('completed').to('executing')
```

Comment:
Name your test subject if you need to reference it explicitly.<br>Prefer change { subject.state }.